docs/release-notes/toc.yml

```diff
@@ -0,0 +1,5 @@
+toc:
+  - file: index.md
+  # - file: known-issues.md
+  # - file: breaking-changes.md
+  # - file: deprecations.md
```

Comment:
Would there be any harm in including these, even though they are empty at the moment?

E.g. the classic java apm agent also includes the [known issues](https://github.com/elastic/apm-agent-java/blob/main/docs/release-notes/known-issues.md) even though they are empty at the moment.

Reply:
I didn't saw your comment before writing mine, I think that if we do that we should at least indicate that they are empty, otherwise it could be quite confusing. The markdown you've linked contains `No known issues` for example.